src/core/zowe/core_for_zowe_sdk/sdk_api.py

```diff
@@ -54,6 +54,8 @@ def __init__(self, profile, default_url, logger_name=__name__):
             self._default_headers["Authorization"] = f"Bearer {self.session.tokenValue}"
         elif self.session.type == session_constants.AUTH_TYPE_TOKEN:
             self._default_headers["Cookie"] = f"{self.session.tokenType}={self.session.tokenValue}"
+        elif self.session.type == session_constants.AUTH_TYPE_CERT_PEM:
+            self.__session_arguments["cert"] = self.session.cert
```

Comment:
This should technically be on another branch (Auth_Type) 😋 

I think it's fine to keep it here because removing it might cause problems later on 😋